spec/cocoa/nsattributedstring_spec.rb

```diff
@@ -54,7 +54,50 @@
       "test\n  ".attrd.strip.should == 'test'.attrd
       "  \n test".attrd.strip.should == 'test'.attrd
     end
+  end
+
+  describe "should allow joining an array of attributed strings" do
+    it "should not care about standard arrays of strings" do
+      joined = ['a', 'b', 'c'].join_attrd
+      joined.should == 'abc'
+      joined.is_a?(NSAttributedString).should == false
+      joined.is_a?(NSString).should == true
```

Comment:
If there's no attributed strings to merge... i.e you're sending it all strings, why wouldn't you just do a `.join.attrd` to turn it into one? Cuts down on overhead and processing time for the loop to just pass it over to the system's `join` method.

But i'm not married to that functionality.